e2e_samples/dataset_versioning/datafactory/README.md

```diff
@@ -0,0 +1,22 @@
+# Overview
+
+This folder is for Data pipeline which copy data from SQL DB to Azure Storage (Delta Lake). We are using Azure Data Factory to implement pipeline.
+
+## How to deploy it
+
+1. Provision Azure Resources by IaC (Terraform)
+1. [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+1. Run command to deploy ([document](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-cli#deploy-local-template-or-bicep-file))
+
+```bash
```

Comment:
Actually, we need to mention to change directories here since the arm_template.json is in ` datafactory/arm_template` but this readme is in `datafactory`

Reply:
Have updated to add path in script `./arm_tempalte/arm_template.json`

Reply:
added correct path in script.